### ISSUE TITLE
Request for libusb [compat] 0.1

### ISSUE THREAD

**Issue by ochrin** [clearlinux/distribution#2309]
Official package name: libusb-compat

License (must be an OSI approved Open Source license): LGPL

Download URL of latest release: https://github.com/libusb/libusb-compat-0.1

Latest release date (must be recent):  Aug 20 2018

Description: A compatibility layer allowing applications written for libusb-0.1 to work
with libusb-1.0

**Comment by ochrin** [comment]
Indeed I needed libusb-0.1 for installing Code Composer Studio (an IDE from Texas Instruments used to develop on their embedded platforms).
In the end I downloaded and compiled libusb-0.1, not the libusb-compat (because at that time I was not successful as I did not know how to add a local library to libraries).
So maybe libusb-compat is enough but I am not sure.
And libusb-0.1 does not seem to be maintained anymore.
